styles.css

```diff
@@ -29,14 +19,27 @@ body {
   justify-content: space-evenly;
   width: 100vw;
   height: 100vh;
-  background: linear-gradient(var(--primary), var(--secondary), var(--third));
+  /* background: linear-gradient(var(--primary), var(--secondary), var(--third)); */
+}
+
+.body-blue-to-yellow-bg {
+    background: linear-gradient(var(--primary), var(--secondary), var(--third));
+}
+
+.body-pacific-dream-bg {
+  background: linear-gradient(#34e89e, #0f3443)
```

Comment:
you could assign these hex values to variables like you are doing on line 26. Give them more descriptive names than primary or secondary. Go for things like "red" or "dark-red".

Reply:
Assigned Hex value variables, and changed the names to fit exactly what they were rather than using terms like primary, secondary, ect.. Changes pushed up